### PR TITLE
fix: linting issue

### PR DESCRIPTION
**Pull request by aundang**
This MR fixes the issue from the last merge.
For more details of the issue check here : https://travis-ci.org/github/GovTechSG/mcf-boilerplate-js/builds/722993060

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "eslint": "4.19.1",
     "eslint-config-google": "0.9.1",
     "lerna": "3.22.1",
-    "mocha": "8.1.2",
+    "mocha": "^5.2.0",
```
Comment (mpr1992) [inline package.json:27]: remove ^

Comment (mpr1992) [reply package.json:27]: why degrade this? from 8.1.2 to 5.2.0

**Review by mpr1992 — CHANGES_REQUESTED** [review]
remove ^  in package.json at line 27.

why degrade the version from 8.1.2 to 5.2.0

File: package.json (hunk 1 above)
Comment (aundang) [reply package.json:27]: Reason for downgrading is due to the issue found here : https://travis-ci.org/github/GovTechSG/mcf-boilerplate-js/jobs/722993061.

Screenshot 2020-09-07 at 1 00 52 PM

This `esm-utils.js` file can only be found on ver 5.2.0.

**Review by mpr1992 — APPROVED** [review]

**Pull request merged**